src/inverted-index.js

```diff
@@ -1,18 +1,25 @@
+/**
+ * Inverted index class
+ */
 class InvertedIndex {
-
+  /**
+   * Inverted index constructor
+   */
   constructor() {
-    //Object to hold the index
+    // Object to hold the index
     this.index = {};
   }
 
-  /** 
+  /**
    * @param{String} words - String to be tokonized
    * @return{Array} cleanContent
    */
   tokenize(words) {
     const cleanContent = words.trim().replace(/-/g, ' ')
```

Comment:
`cleanContent` is redundant. The variable is declared only to be returned immediately. 
What do you think about just returning the values to be returned?